native/src/components/LayoutedScrollView.tsx

```diff
@@ -15,6 +15,7 @@ const LayoutedScrollView = (props: LayoutedScrollViewProps): ReactElement => {
     <Layout>
       <ScrollView
         keyboardShouldPersistTaps='always'
+        nestedScrollEnabled
```

Comment:
why do we need that?

Reply:
@steffenkleinle yep, this exact error

Reply:
But do you have a resource that says that this `nestedScrollEnabled` fixes the error? Or did you just try things until you found a solution?

Reply:
@steffenkleinle many developers mentioned this solution when I was searching for an answer why

Reply:
[Here is the link](https://github.com/facebook/react-native/issues/31697)